src/components/presentational/TimeSeriesChart/TimeSeriesChart.stories.tsx

```diff
@@ -342,4 +344,52 @@ export const loading: Story = {
         intervalStart: new Date(),
         tooltip
     },
+};
+
+const generateSteps = (date: Date) => {
+    let steps: { timestamp: Date, value: number }[] = [];
+
+    let timestamp = add(startOfDay(date), { minutes: 15 });
+    while (isSameDay(timestamp, date)) {
+        let newValue = Math.round((Math.random() * 200) + 20);
```

Comment:
Nit: Why not use one of the existing random number generator functions in this same module?

Reply:
I have updated the story to use the existing random intraday data function to generate the steps data.